src/components/spinner/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+import Loader from 'react-loader';
+import SC from './styled';
+
+export default class Spinner extends Component {
```

Comment:
Siden komponenten er stateless kan du forenkle og lage som en functional component:
const Spinner: React.FC<Props> = () => (
<SC.Spinner> osv
);

Reply:
Ah, ok Øyvind, tusen takk for tips! :-)

Reply:
Ok!